### DISCUSSION TITLE
[Feature] Edit Capture Date and Write to Exif

### DISCUSSION THREAD

**Discussion by AngelaDMerkel** [immich-app/immich#1940] [Feature Request]
### The feature

A common use case is editing the date or time of images which have been sent over messaging applications like WhatsApp. These are stripped of metadata in transit and when backed are chronologically incorrect. Being able to edit the date metadata and having it written to original files would both improve user experience and ensure that these files durably retain any time put into editing their times and dates. The chronic dissatisfaction with Takeout is often related to metadata not being written into the originals, and I believe this feature would be well received. 

Ideally there would be support for including the timezone information too. 

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by alextran1502** [comment]
Duplicated of #1696
